### PR TITLE
Replace DevToolTime with Discord in search footer

### PR DESCRIPTION
**Pull request by jdorfman**
Signed-off-by: Justin Dorfman <justin.dorfman@sourcegraph.com>

Replaces the DevToolTime logo/ad with a Discord server invite.

[Slack discussion](https://sourcegraph.slack.com/archives/C015P61EXV1/p1660321223979559)

## Test plan

- Is the svg logo optimized
- Make sure link works
- Grammar check
- Run by Fabiana (branding)

## App preview:

- [Web](https://sg-web-jd-update-search-footer.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-kucszqcvsv.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by jdorfman** [comment]
@fabicastp This will replace this:
Image 2022-08-12 at 12 41 50 PM

With this:
Image 2022-08-12 at 12 42 28 PM

**Review by limitedmage — APPROVED** [review]
LGTM from the dev side 👍

File: client/web/src/search/home/SearchPageFooter.tsx [hunk 1]
```diff
@@ -101,18 +101,18 @@ export const SearchPageFooter: React.FunctionComponent<
                 ))}
                 <li>
                     <Link
-                        to="https://info.sourcegraph.com/dev-tool-time"
+                        to="https://srcgr.ph/discord-server"
                         className={styles.devToolTimeWrapper}
```
Comment (limitedmage) [inline client/web/src/search/home/SearchPageFooter.tsx:105]: Very minor but may want to replace this and other CSS class that say `devToolTime` with something more relevant here (maybe `discord`?)

File: client/web/src/search/home/SearchPageFooter.tsx [hunk 2]
```diff
@@ -101,18 +101,18 @@ export const SearchPageFooter: React.FunctionComponent<
...
                         onClick={logDevelopmentToolTimeClicked}
                     >
                         <img
-                            src={`${assetsRoot}/img/devtooltime-logo.svg`}
```
Comment (limitedmage) [inline client/web/src/search/home/SearchPageFooter.tsx:109]: We can delete this image if it's not used anywhere anymore.

File: client/web/src/search/home/SearchPageFooter.tsx (hunk 1 above)
Comment (jdorfman) [reply client/web/src/search/home/SearchPageFooter.tsx:105]: @limitedmage good catch. I'm not sure what to do. Can I just rename it or do I need to update the class in a CSS file?

File: client/web/src/search/home/SearchPageFooter.tsx (hunk 2 above)
Comment (jdorfman) [reply client/web/src/search/home/SearchPageFooter.tsx:109]: done

**Comment by jdorfman** [comment]
@limitedmage can you help me with the CSS 🙏

**Comment by danieldides** [comment]
I'm not sure if it matters but that is the [old discord logo](https://support.discord.com/hc/en-us/articles/1500009438682-A-Fresh-New-Look-to-Celebrate-Our-6th-Birthday) in the screenshot.

**Comment by jdorfman** [comment]
@danieldides yeah I was emphasizing what this PR would replace. In hindsight, I can see how it could be confusing.

**Comment by jdorfman** [comment]
@danieldides @limitedmage what are the final steps to get this merged?

**Comment by danieldides** [comment]
@jdorfman It looks like it's failing a check in buildkite
> Checked 2 files and found SVG optimizations. Please run 'yarn optimize-svg-assets ui/assets/img' and commit the result.

**Comment by jdorfman** [comment]
@danieldides it keeps failing even those I'm getting this:

```
$ yarn optimize-svg-assets ui/assets/img

yarn run v1.22.4
$ svgo --multipass --config="./svgo.config.js" ui/assets/img
Processing directory 'ui/assets/img':


codemonitoring-notify-dark.svg:
Done in 40 ms!
1.136 KiB - 0% = 1.136 KiB

codemonitoring-search-dark.svg:
Done in 20 ms!
0.986 KiB - 0% = 0.986 KiB

codemonitoring-notify-light.svg:
Done in 26 ms!
1.129 KiB - 0% = 1.129 KiB

customer-logos-dark.svg:
Done in 92 ms!
15.871 KiB - 0% = 15.871 KiB

codemonitoring-illustration-light.svg:
Done in 109 ms!
46.549 KiB - 0% = 46.549 KiB

discord-footer-logo.svg:
Done in 25 ms!
1.051 KiB - 0% = 1.051 KiB

homepage-hero-dark.svg:
Done in 911 ms!
179.215 KiB - 0% = 179.215 KiB

logo-chrome.svg:
Done in 27 ms!
3.286 KiB - 0% = 3.286 KiB

codemonitoring-illustration-dark.svg:
Done in 103 ms!
47.935 KiB - 0% = 47.935 KiB

customer-logos-light.svg:
Done in 76 ms!
15.87 KiB - 0% = 15.87 KiB

codemonitoring-search-light.svg:
Done in 4 ms!
0.983 KiB - 0% = 0.983 KiB

sourcegraph-mark.svg:
Done in 2 ms!
0.528 KiB - 0% = 0.528 KiB

homepage-hero-light.svg:
Done in 750 ms!
173.888 KiB - 0% = 173.888 KiB

notepad-illustration-light.svg:
Done in 26 ms!
6.279 KiB - 0% = 6.279 KiB

sourcegraph-logo-dark.svg:
Done in 10 ms!
4.795 KiB - 0% = 4.795 KiB

sourcegraph-logo-light.svg:
Done in 14 ms!
4.795 KiB - 0% = 4.795 KiB

notepad-illustration-dark.svg:
Done in 18 ms!
6.498 KiB - 0% = 6.498 KiB

logo-firefox.svg:
Done in 414 ms!
211.686 KiB - 0% = 211.686 KiB
✨  Done in 3.54s.
```

File: enterprise/internal/insights/resolvers/aggregates_resolvers.go [hunk 3]
```diff
@@ -16,7 +16,6 @@ type searchAggregateResolver struct {
 
 func (r *searchAggregateResolver) ModeAvailability(ctx context.Context) []graphqlbackend.AggregationModeAvailabilityResolver {
 	resolvers := []graphqlbackend.AggregationModeAvailabilityResolver{}
-	for _, mode := range types.SearchAggregationModes {
```
Comment (chwarwick) [inline enterprise/internal/insights/resolvers/aggregates_resolvers.go:19]: The changes in this file look like they are accidentally included.

**Review by chwarwick — CHANGES_REQUESTED** [review]

**Comment by jdorfman** [comment]
Opening up a new PR, don't ask.

**Pull request closed without merging**